website/docs/releases/2025/v2025.4.md

```diff
@@ -17,7 +17,7 @@ slug: "/releases/2025.4"
 
 ## Breaking changes
 
-- **Reputation score limit**: The default value for the new limits on Reputation score is between `-5` and `5`. This might break some current setups which count on the possibility of scores decreasing or increasing beyond these limits. You can set your custom limits under **System > Settings**.
+- **Reputation score limit**: The defaults for the new upper and lower limits on Reputation score are `-5` and `5`. This might break custom policies that rely on the reputation scores decreasing or increasing beyond these limits. You can set your custom limits under **System > Settings**.
```

Comment:
```suggestion
- **Reputation score limit**: The default values for the new upper and lower limits on Reputation score are `-5` and `5`. This might break custom policies that rely on the reputation scores decreasing or increasing beyond these limits. You can set your custom limits under **System > Settings**.
```